Shipfinity.DTOs/SellerDTO's/SellerLoginResponseDto.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shipfinity.DTOs.SellerDTO_s
+{
+    public class SellerLoginResponseDto
+    {
+        public int Id { get; set; }
+        public string Username { get; set; }
+        public string Email { get; set; }
+        public string Name { get; set; }
```

Comment:
add the role aswell `public string Role { get; set; }`

Reply:
done